### ISSUE TITLE
Tests execution is failing

### ISSUE THREAD

**Issue by lucaskanashiro** [biola/Voight-Kampff#40]
Hi,

The test execution via `rspec` is failing. Follow the steps below to reproduce it:

```
$ bundle install --path vendor/bundle
$ bundle exec rspec
``` 

The  output of the last command is the following:

```
An error occurred while loading ./spec/controllers/replicants_controller_spec.rb.
Failure/Error: require 'rspec/rails'

NameError:
  uninitialized constant Capybara::VERSION
# ./spec/spec_helper.rb:7:in `<top (required)>'
# ./spec/controllers/replicants_controller_spec.rb:1:in `require'
# ./spec/controllers/replicants_controller_spec.rb:1:in `<top (required)>'

An error occurred while loading ./spec/lib/voight_kampff/rack_request_spec.rb.
Failure/Error: Combustion.initialize! :action_controller

RuntimeError:
  Application has been already initialized.
# ./vendor/bundle/ruby/2.5.0/gems/railties-5.2.3/lib/rails/application.rb:360:in `initialize!'
# ./vendor/bundle/ruby/2.5.0/gems/railties-5.2.3/lib/rails/railtie.rb:190:in `public_send'
# ./vendor/bundle/ruby/2.5.0/gems/railties-5.2.3/lib/rails/railtie.rb:190:in `method_missing'
# ./vendor/bundle/ruby/2.5.0/gems/combustion-0.9.1/lib/combustion.rb:32:in `initialize!'
# ./spec/spec_helper.rb:5:in `<top (required)>'
# ./vendor/bundle/ruby/2.5.0/gems/activesupport-5.2.3/lib/active_support/dependencies.rb:291:in `require'
# ./vendor/bundle/ruby/2.5.0/gems/activesupport-5.2.3/lib/active_support/dependencies.rb:291:in `block in require'
# ./vendor/bundle/ruby/2.5.0/gems/activesupport-5.2.3/lib/active_support/dependencies.rb:257:in `load_dependency'
# ./vendor/bundle/ruby/2.5.0/gems/activesupport-5.2.3/lib/active_support/dependencies.rb:291:in `require'
# ./spec/lib/voight_kampff/rack_request_spec.rb:1:in `<top (required)>'
# ./vendor/bundle/ruby/2.5.0/gems/activesupport-5.2.3/lib/active_support/dependencies.rb:285:in `load'
# ./vendor/bundle/ruby/2.5.0/gems/activesupport-5.2.3/lib/active_support/dependencies.rb:285:in `block in load'
# ./vendor/bundle/ruby/2.5.0/gems/activesupport-5.2.3/lib/active_support/dependencies.rb:257:in `load_dependency'
# ./vendor/bundle/ruby/2.5.0/gems/activesupport-5.2.3/lib/active_support/dependencies.rb:285:in `load'

An error occurred while loading ./spec/lib/voight_kampff/test_spec.rb.
Failure/Error: Combustion.initialize! :action_controller

RuntimeError:
  Application has been already initialized.
# ./vendor/bundle/ruby/2.5.0/gems/railties-5.2.3/lib/rails/application.rb:360:in `initialize!'
# ./vendor/bundle/ruby/2.5.0/gems/railties-5.2.3/lib/rails/railtie.rb:190:in `public_send'
# ./vendor/bundle/ruby/2.5.0/gems/railties-5.2.3/lib/rails/railtie.rb:190:in `method_missing'
# ./vendor/bundle/ruby/2.5.0/gems/combustion-0.9.1/lib/combustion.rb:32:in `initialize!'
# ./spec/spec_helper.rb:5:in `<top (required)>'
# ./vendor/bundle/ruby/2.5.0/gems/activesupport-5.2.3/lib/active_support/dependencies.rb:291:in `require'
# ./vendor/bundle/ruby/2.5.0/gems/activesupport-5.2.3/lib/active_support/dependencies.rb:291:in `block in require'
# ./vendor/bundle/ruby/2.5.0/gems/activesupport-5.2.3/lib/active_support/dependencies.rb:257:in `load_dependency'
# ./vendor/bundle/ruby/2.5.0/gems/activesupport-5.2.3/lib/active_support/dependencies.rb:291:in `require'
# ./spec/lib/voight_kampff/test_spec.rb:1:in `<top (required)>'
# ./vendor/bundle/ruby/2.5.0/gems/activesupport-5.2.3/lib/active_support/dependencies.rb:285:in `load'
# ./vendor/bundle/ruby/2.5.0/gems/activesupport-5.2.3/lib/active_support/dependencies.rb:285:in `block in load'
# ./vendor/bundle/ruby/2.5.0/gems/activesupport-5.2.3/lib/active_support/dependencies.rb:257:in `load_dependency'
# ./vendor/bundle/ruby/2.5.0/gems/activesupport-5.2.3/lib/active_support/dependencies.rb:285:in `load'

An error occurred while loading ./spec/lib/voight_kampff_spec.rb.
Failure/Error: Combustion.initialize! :action_controller

RuntimeError:
  Application has been already initialized.
# ./vendor/bundle/ruby/2.5.0/gems/railties-5.2.3/lib/rails/application.rb:360:in `initialize!'
# ./vendor/bundle/ruby/2.5.0/gems/railties-5.2.3/lib/rails/railtie.rb:190:in `public_send'
# ./vendor/bundle/ruby/2.5.0/gems/railties-5.2.3/lib/rails/railtie.rb:190:in `method_missing'
# ./vendor/bundle/ruby/2.5.0/gems/combustion-0.9.1/lib/combustion.rb:32:in `initialize!'
# ./spec/spec_helper.rb:5:in `<top (required)>'
# ./vendor/bundle/ruby/2.5.0/gems/activesupport-5.2.3/lib/active_support/dependencies.rb:291:in `require'
# ./vendor/bundle/ruby/2.5.0/gems/activesupport-5.2.3/lib/active_support/dependencies.rb:291:in `block in require'
# ./vendor/bundle/ruby/2.5.0/gems/activesupport-5.2.3/lib/active_support/dependencies.rb:257:in `load_dependency'
# ./vendor/bundle/ruby/2.5.0/gems/activesupport-5.2.3/lib/active_support/dependencies.rb:291:in `require'
# ./spec/lib/voight_kampff_spec.rb:1:in `<top (required)>'
# ./vendor/bundle/ruby/2.5.0/gems/activesupport-5.2.3/lib/active_support/dependencies.rb:285:in `load'
# ./vendor/bundle/ruby/2.5.0/gems/activesupport-5.2.3/lib/active_support/dependencies.rb:285:in `block in load'
# ./vendor/bundle/ruby/2.5.0/gems/activesupport-5.2.3/lib/active_support/dependencies.rb:257:in `load_dependency'
# ./vendor/bundle/ruby/2.5.0/gems/activesupport-5.2.3/lib/active_support/dependencies.rb:285:in `load'
No examples found.


Finished in 0.00018 seconds (files took 0.6543 seconds to load)
0 examples, 0 failures, 4 errors occurred outside of examples
```

This problem has been reported in the Debian BTS twice:

* https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=921901
* https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=925565

Thanks for your work!

**Comment by adamcrown** [comment]
Unfortunately, I'm not able to reproduce this error. The spec suite is passing for me locally and on Travis CI. I did upgrade some of the dependencies, as they were pretty out of date. Could you try again and see if that fixes it for you.

The strangest thing about this is that it complains about Capybara, but nothing in the test suite of this gem should be using Capybara at all.

**Comment by lucaskanashiro** [comment]
Sorry for bothering, I just made a quick test and forwarded the Debian bug reports to you. Now, I made an extra verification and noticed that this is some kind of conflict with 'rails' dependencies tree in Debian. Despite I was installing packages from rubygems (not Debian packages) after I removed the 'rails' Debian package and its dependencies from my system the tests are working fine:

```
$ bundle exec rspec
............................................

Finished in 0.08145 seconds (files took 0.55997 seconds to load)
44 examples, 0 failures
```
Will investigate it further in the Debian side.
